### PR TITLE
Adding 'distance' return field to the 'polylabel' package

### PR DESCRIPTION
**Pull request by JonLoesch**
This was added to library in June 2020 -- see https://github.com/mapbox/polylabel/pull/61

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)

- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/mapbox/polylabel/pull/61
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by JonLoesch** [comment]
I've never contributed to DefinitelyTyped before, so please let me know if I'm making any dumb mistakes.

File: types/polylabel/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for polylabel 1.0
```
Comment (JonLoesch) [inline types/polylabel/index.d.ts:1]: There's an item in the PR checklist:

>  If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

I *think* this is what's meant?

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## polylabel ([<kbd>unpkg</kbd>](https://unpkg.com/browse/polylabel@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'polylabel'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4cd1db722c2878fcfc5de8eb021e8684da10c248
</p>

**Comment by JonLoesch** [comment]
> 1. The declaration doesn't match the JavaScript module 'polylabel'. Reason:
>    The JavaScript module can be called or constructed, but the declaration module cannot.

Responding to the bot.

I'm not an expert in this stuff, so I could be misinterpreting something.  The module in question exports using both `export =` and `export default`.  ([source](https://github.com/mapbox/polylabel/blob/07c112091b4c9ffeb412af33c575133168893b4a/polylabel.js#L7)).  Not sure what DefinitelyTyped would want in this scenario, not sure if there's any fancy TS tech to support this.  But considering that I'm not sure, and there's also a previous pull request (#19869) changing exactly this, I'm inclined to just not touch it.

File: types/polylabel/index.d.ts (hunk 1 above)
Comment (MysteryBlokHed) [reply types/polylabel/index.d.ts:1]: Yep, that's all you need to do 😊

**Review by MysteryBlokHed — APPROVED** [review]
LGTM, but a package owner will need to sign off before it can be merged. As for the `export default` vs `export =`, I guess I'd just leave it as-is if someone else changed it before.

**Comment by JonLoesch** [comment]
Ready to merge

**Pull request merged**